transit_odp/organisation/csv/overall.py

```diff
@@ -89,6 +95,54 @@
     "string_lines",
 )
 
+DATACATALOGUE_ATTRIBUTE_FIELDS = (
+    "fares_metadata_id",
+    "xml_file_name",
+    "national_operator_code",
+    "line_name",
+)
+
+
+def get_fares_data():
+    """
+    Returns list of fares information (xml file name, NOCs and line name)
+    when dataset_id in DatasetRevision table match
+    fares_metadata_id in DataCatalogueMetaData table.
+    """
+    dataset_ids_dict = (
+        DataCatalogueMetaData.objects.filter(
+            fares_metadata_id__in=FaresMetadata.objects.all().values_list(
```

Comment:
consider making this a subquery, you can flatten the values list with `flat=True`